_release/management-interface.yaml

```diff
@@ -0,0 +1,2 @@
+port: 80
+cptv-dir: "/var/spool/cptv/"
```

Comment:
Eventually we want management-interface, thermal-recorder and thermal-uploader to all read this from the same place, but this is fine for now.